src/app.js

```diff
@@ -68,12 +74,18 @@ Vue.use(VueMeta, {
   debounceWait: 500
 })
 
+Vue.use(VueHighlightJS, {
+  languages: {
+    json,
+    yaml
+  }
+})
+
 Vue.prototype.$globalApolloQueries = {}
 
 // Vue Global Error Handler
 Vue.config.errorHandler = function(error, vm, trace) {
-  if (error?.message?.includes("Cannot read property '_observe' of null"))
-    return
+  if (error?.message?.includes("'_observe'")) return
```

Comment:
no, this has been here, I just made the catch more inclusive because I was seeing basically the same error worded a little differently. Spoke with Nicholas briefly about trying to remove this but decided it was a task for another day